terraform/variables.tf

```diff
@@ -251,6 +251,12 @@ variable "enable_lb_logging" {
   EOT
 }
 
+variable "log_retention_period" {
+  type        = number
+  default     = 30
```

Comment:
I think we should make the default 14 - we should also note it impacts ALL resources in the project

Reply:
Updated and updated release notes.